eap73-rhel8-payg-vmss/scripts/jbosseap-setup-redhat.sh

```diff
@@ -147,12 +146,32 @@ echo -e "JAVA_OPTS=\"\$JAVA_OPTS -Djboss.jgroups.azure_ping.container=$CONTAINER
 echo "Start JBoss-EAP service"                | log; flag=${PIPESTATUS[0]}
 echo "systemctl enable eap7-standalone.service" | log; flag=${PIPESTATUS[0]}
 systemctl enable eap7-standalone.service      | log; flag=${PIPESTATUS[0]}
+####################### 
+
+###################### Editing eap7-standalone.services and adding the following lines
+echo "Adding - After=syslog.target network.target NetworkManager-wait-online.service" | log; flag=${PIPESTATUS[0]}
+sed -i 's/After=syslog.target network.target/After=syslog.target network.target NetworkManager-wait-online.service/' /etc/systemd/system/multi-user.target.wants/eap7-standalone.service
+echo "Adding - Wants=NetworkManager-wait-online.service \nBefore=httpd.service" | log; flag=${PIPESTATUS[0]}
+sed -i 's/Before=httpd.service/Wants=NetworkManager-wait-online.service \nBefore=httpd.service/' /etc/systemd/system/multi-user.target.wants/eap7-standalone.service
+echo "systemctl daemon-reload" | log; flag=${PIPESTATUS[0]}
+systemctl daemon-reload
 
 echo "systemctl restart eap7-standalone.service"| log; flag=${PIPESTATUS[0]}
-systemctl restart eap7-standalone.service     | log; flag=${PIPESTATUS[0]}
+systemctl restart eap7-standalone.service       | log; flag=${PIPESTATUS[0]}
 echo "systemctl status eap7-standalone.service" | log; flag=${PIPESTATUS[0]}
-systemctl status eap7-standalone.service      | log; flag=${PIPESTATUS[0]}
-####################### 
+systemctl status eap7-standalone.service        | log; flag=${PIPESTATUS[0]}
+######################
+
+####################### Starting cron job
+echo "systemctl restart eap7-standalone.service" >> /bin/jbossservice.sh
+chmod +x /bin/jbossservice.sh
+
+yum install cronie cronie-anacron | log; flag=${PIPESTATUS[0]}
+service crond start | log; flag=${PIPESTATUS[0]}
+chkconfig crond on | log; flag=${PIPESTATUS[0]}
+echo "@reboot sleep 90 && /bin/jbossservice.sh" >>  /var/spool/cron/root
+chmod 600 /var/spool/cron/root
+#######################
```

Comment:
I believe adding Cron job shouldn't be needed as `eap7-standalone.service` is already waiting for IP address to be assigned with the changes into `eap7-standalone.services` file. Actually this is adding one more restart of EAP 7 after boot which is not desired.